pkg/ebpf/tracer.go

```diff
@@ -173,37 +177,16 @@ func (t *Tracer) expvarStats() {
 			continue
 		}
 
-		if tracerStats, ok := stats["tracer"]; ok {
-			for metric, val := range tracerStats.(map[string]int64) {
-				currVal := &expvar.Int{}
-				currVal.Set(val)
-				probeExpvar.Set(snakeToCapInitialCamel(metric), currVal)
-			}
-		}
-
-		if ebpfStats, ok := stats["ebpf"]; ok {
-			for metric, val := range ebpfStats.(map[string]int64) {
-				currVal := &expvar.Int{}
-				currVal.Set(val)
-				probeExpvar.Set(fmt.Sprintf("Ebpf%s", snakeToCapInitialCamel(metric)), currVal)
-			}
-		}
-
-		if states, ok := stats["state"]; ok {
-			if telemetry, ok := states.(map[string]interface{})["telemetry"]; ok {
-				for metric, val := range telemetry.(map[string]int64) {
-					currVal := &expvar.Int{}
-					currVal.Set(val)
-					probeExpvar.Set(snakeToCapInitialCamel(metric), currVal)
+		for name, stat := range stats {
+			if name == "state" {
+				if telemetry, ok := stat.(map[string]interface{})["telemetry"]; ok {
```

Comment:
sorry last comment can we add a `GetTelemetry` like func to get this instead of retrieving from `GetStats` ? this way we could avoid this edge case.

And we could call `GetTelemetry` in `GetStats`